bundles/org.openhab.binding.jeelink/src/main/java/org/openhab/binding/jeelink/internal/lacrosse/LgwSensorDefinition.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.jeelink.internal.lacrosse;
+
+import org.eclipse.smarthome.core.thing.Thing;
+import org.openhab.binding.jeelink.internal.JeeLinkBindingConstants;
+import org.openhab.binding.jeelink.internal.JeeLinkReadingConverter;
+import org.openhab.binding.jeelink.internal.JeeLinkSensorHandler;
+import org.openhab.binding.jeelink.internal.SensorDefinition;
+
+/**
+ * Sensor definition of a sensor directly connected to a LGW.
+ *
+ * @author Volker Bier - Initial contribution
+ */
+public class LgwSensorDefinition extends SensorDefinition<LgwReading> {
```

Comment:
see above

Reply:
LgwSensorDefinition.java:[45,1443]: Illegal redefinition of parameter thing, inherited method from SensorDefinition<LgwReading> does not constrain this parameter